mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/SurefireProviderResolver.java

```diff
@@ -0,0 +1,85 @@
+package org.arquillian.smart.testing.mvn.ext.dependencies;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.maven.model.Dependency;
+import org.arquillian.smart.testing.configuration.Configuration;
+import org.arquillian.smart.testing.logger.Log;
+import org.arquillian.smart.testing.logger.Logger;
+import org.arquillian.smart.testing.known.surefire.providers.KnownProvider;
+
+import static org.arquillian.smart.testing.known.surefire.providers.KnownProvider.JUNIT_4;
+import static org.arquillian.smart.testing.known.surefire.providers.KnownProvider.JUNIT_47;
+import static org.arquillian.smart.testing.known.surefire.providers.KnownProvider.JUNIT_5;
+import static org.arquillian.smart.testing.known.surefire.providers.KnownProvider.TESTNG;
+
+class SurefireProviderResolver {
+
+    private final Logger log = Log.getLogger();
```

Comment:
I think that in almost all cases (or at least the ones I have seen) `Logger` is used as a static field.